src/site/components/up_to_top_button.html

```diff
@@ -0,0 +1,96 @@
+                        if(!threeFourthsContainer || !upToTopButton || !footer) return 'NODE_MISSING_CLASS' 
+                    }
+
+                    function scrollListener() {
+                        // Responsible for toggling animation classes 
+                        var scrollTop = (window.pageYOffset !== undefined) ? window.pageYOffset : (document.documentElement || document.body.parentNode || document.body).scrollTop;
+                        var buttonTransitionIn = 'vsa-top-button-transition-in'
+                        var buttonTransitionOut = 'vsa-top-button-transition-out'
+                        var buttonContainerRelative = 'vsa-top-button-container-relative'
+                        var buttonTransitionReset = 'vsa-top-button-transition-reset'
+                        
+                        
+                        if(scrollTop > 600 && !doesElHaveClass(upToTopButton, buttonTransitionIn)) {
+                            upToTopButton.classList.add(buttonTransitionIn)
+                            upToTopButton.classList.remove(buttonTransitionOut)
+                        } else if (scrollTop < 600 && doesElHaveClass(upToTopButton, buttonTransitionIn)) {
+                            upToTopButton.classList.add(buttonTransitionOut)
+                            upToTopButton.classList.remove(buttonTransitionIn)
+                        }
+
+                        if(isScrolledIntoView(footer)) {
+                            container.classList.add(buttonContainerRelative)
+                            threeFourthsContainer.classList.add(buttonContainerRelative)
+                            upToTopButton.classList.add(buttonTransitionReset)
+                        }
+                        else if(doesElHaveClass(container, buttonContainerRelative)) {
+                            container.classList.remove(buttonContainerRelative)
+                            threeFourthsContainer.classList.remove(buttonContainerRelative)
+                            upToTopButton.classList.remove(buttonTransitionReset)
+                        }
+
+                        function doesElHaveClass(el, className) {
+                            var hasClass = el.classList.value.indexOf(className) > -1
+                            return hasClass
+                        }
+
+                        function isScrolledIntoView(el) {
+                            var rect = el.getBoundingClientRect();
+                            var elemTop = rect.top;
+
+                            // Only partially || completely visible elements return true
+                            var isVisible = (elemTop >= 0) && (elemTop <= window.innerHeight);
+                            return isVisible;
+                        }
+                    }
+                    
+                    function navigateToTop() {
+                        return window.scrollTo(0,0)
+                    }
```

Comment:
Can I ask that we move this into the `src/applications/static-pages`? I think we could follow these steps -

1. Create `src/applications/static-pages/scroll-top-button/index.js` (name it what you think is best). It should export a function to serve as the entry point.
    - `src/applications/static-pages/social-share-links.js` is an example of the same code flow
1. Import that function and execute it from `src/applications/static-pages/static-pages-entry.js`
1. Changes to this code can be hot-reloaded using the command `yarn watch --env.entry=static-pages`. 
    - If you aren't updating the template anymore, you can just `yarn build:content` a final time, then only run the watch command, bc you will only need Webpack
1. Run linter, take advantage of ES6, etc

Reply:
Yeah, I think we can. I'm totally up for suggestions. It will help me learn more of the how to repo works.

Reply:
@ncksllvn Ready to go again.